examples/nnp_training_ignite.py

```diff
@@ -102,13 +101,9 @@ def atomic():
 
 ###############################################################################
 # Now load training and validation datasets into memory.
-training = torchani.data.BatchedANIDataset(
-    training_path, consts.species_to_tensor, batch_size, device=device,
-    transform=[energy_shifter.subtract_from_dataset])
-
-validation = torchani.data.BatchedANIDataset(
-    validation_path, consts.species_to_tensor, batch_size, device=device,
-    transform=[energy_shifter.subtract_from_dataset])
+validation, training = torchani.data.load_ani_dataset(
```

Comment:
Why not make the order consistent "training, validation" and then keep split=[0.8, None] ?

Reply:
fixed